### ISSUE TITLE
Exception error when activating the user profile

### ISSUE THREAD

**Issue by xchs** [contao/contao#130]
**Affected version(s)**

4.6+ (et al.)

**Description**  

The following exception error is generated (and tracked and reported by Sentry) when the back end module "_User profile_" is activated in the user's settings:

https://sentry.io/share/issue/17bb6d37f2584af5bafe9b4f9c4ec3b2/

```
ErrorException: Invalid argument supplied for foreach()

#17 vendor/contao/core-bundle/src/EventListener/BackendMenuListener.php(68): handleError
#16 vendor/contao/core-bundle/src/EventListener/BackendMenuListener.php(68): onBuild
#15 vendor/symfony/event-dispatcher/EventDispatcher.php(212): doDispatch
#14 vendor/symfony/event-dispatcher/EventDispatcher.php(44): dispatch
#13 vendor/contao/core-bundle/src/Menu/BackendMenuBuilder.php(46): create
#12 vendor/contao/core-bundle/src/Resources/contao/controllers/BackendMain.php(272): output
#11 vendor/contao/core-bundle/src/Resources/contao/controllers/BackendMain.php(164): run
#10 vendor/contao/core-bundle/src/Controller/BackendController.php(48): mainAction
#9 vendor/symfony/http-kernel/HttpKernel.php(149): handleRaw
#8 vendor/symfony/http-kernel/HttpKernel.php(66): handle
#7 vendor/symfony/http-kernel/Kernel.php(188): handle
#6 vendor/symfony/http-kernel/HttpCache/SubRequestHandler.php(85): handle
#5 vendor/symfony/http-kernel/HttpCache/HttpCache.php(448): forward
#4 vendor/symfony/framework-bundle/HttpCache/HttpCache.php(57): forward
#3 vendor/symfony/http-kernel/HttpCache/HttpCache.php(238): pass
#2 vendor/symfony/http-kernel/HttpCache/HttpCache.php(183): handle
#1 vendor/friendsofsymfony/http-cache/src/SymfonyCache/EventDispatchingHttpCache.php(98): handle
#0 web/app.php(54): null
```

![user_settings](https://user-images.githubusercontent.com/1218766/46909636-fb3b8b80-cf35-11e8-8c52-4050704b62d7.png)

Why is there the entry "_User profile_" in the "_Account manager_" options at all? Every user can access his profile settings via the dropdown menu in the upper right corner anyway.

![user_be](https://user-images.githubusercontent.com/1218766/46909698-f3301b80-cf36-11e8-9b82-fef9e991516e.png)

**How to reproduce**  

To reproduce in the official demo:
* Log into the back end
* _Account manager_ > _Users_: Edit the user's settings for e.g. James Wilson
* _Allowed modules_ > _Back end modules_ > _Account manager_: Enable the "_User profile_" option
* Save the record
* Switch to the user's back end

**Comment by leofeyer** [comment]
The user profile should not appear in the list of allowable modules.

**Comment by leofeyer** [comment]
Fixed in 5de057c42c1df475ebb51ef06fb98f0f4eebfea8.
